Public/Src/FrontEnd/MsBuild/PipConstructor.cs

```diff
@@ -455,6 +455,11 @@ private void SetProcessEnvironmentVariables(IReadOnlyDictionary<string, string>
             processBuilder.AddOutputFile(logDirectory.Combine(PathTable, "msbuild.err"), FileExistence.Optional);
             processBuilder.AddOutputFile(logDirectory.Combine(PathTable, "msbuild.prf"), FileExistence.Optional);
 
+            if (m_resolverSettings?.EnableBinLogTracing == true)

```

Comment:
Why use the null conditional operator? According to line 438, m_resolverSettings != null

Reply:
:+1:

---
In reply to: [273752434](https://github.com/Microsoft/BuildXL/pull/116#discussion_r273752434) [](ancestors = 273752434)

Reply:
No real reasons, copy and paste :)

---
In reply to: [273753511](https://github.com/Microsoft/BuildXL/pull/116#discussion_r273753511) [](ancestors = 273753511,273752434)